### ISSUE TITLE
Implement RPI Catalog API

### ISSUE THREAD

**Issue by Bad-Science** [YACS-RCOS/yacs#30]
Currently we are parsing and scraping catalog data from multiple sources. There exists an API for the RPI catalog and we should be taking advantage of that. Information on this API will be posted here as it becomes available.

**Comment by aosdict** [comment]
This may get kicked down the road and not be available for the initial RPI release. We are currently waiting on RPI for this and there's not really any work we can do on it until then.

**Comment by Bad-Science** [comment]
a future implementation of #22 is now folded into this issue. The catalog api rpi adapter should implement all methods in lib/catalog/abstract_adapter.rb. Because this is blocked by what is to us "acts of god" I am removing this from the milestone.

**Comment by aosdict** [comment]
Acts of God? More like acts of red tape.

**Comment by copperwater** [comment]
Update: We've got access to the Acalog system but it appears to be virtually unused, with a lot of important functionality being undocumented.

**Comment by Bad-Science** [comment]
Spring cleaning! This issue was solved by https://github.com/yacs-rcos/adapter-acalog-rpi
